src/logging_vnode.erl

```diff
@@ -115,7 +131,14 @@ init([Partition]) ->
     LogFile = integer_to_list(Partition),
     {ok, Ring} = riak_core_ring_manager:get_my_ring(),
     GrossPreflists = riak_core_ring:all_preflists(Ring, ?N),
-    Preflists = lists:filter(fun(X) -> preflist_member(Partition, X) end, GrossPreflists),
+    Preflists = lists:foldl(fun(X, Filtered) ->
+                                    case preflist_member(Partition, X) of
```

Comment:
This change seems wrong: Filter should be more performant than fold (O(N) vs O(Nˆ2)) due to the append to the list.